README.md

```diff
@@ -178,3 +305,13 @@ make test-all RULES=rules/libcurl.json
 # Test all rules on all OSs
 make test-all
 ```
+
+### Schema
+
+The JSON schema is defined in the file [`schema.json`](schema.json). Do not
+modify this file directly, since it is automatically generated. Instead, modify
+`schema.template.json` and then run `npm run generate-schema`. The
+`generate-schema` target is automatically run when running `npm test`.
+
+If you need to modify the distros and/or versions supported in the schema definitions, 
+modify `generate-schema.js`.
```

Comment:
Should this reference `systems.js` instead?

Reply:
Nice catch, I forgot about that. I also updated the schema reference to point to that for supported OS versions.